app/routes/user.js

```diff
@@ -113,6 +113,7 @@ module.exports = [
     method: 'DELETE',
     path: '/user/{userId?}',
     options: {
+      auth: { scope: [scopes.admin] },
```

Comment:
I think we can add 'POST /user' and 'POST /users' under admin,

Reply:
these were done previously, a week or so ago